client/app/dashboard/overview/overview.component.js

```diff
@@ -25,6 +25,16 @@ function updateTemperatures($scope,temp) {
 
 }
 
+function updateStates(bms,car)
+{
+  if(bms){
+    angular.element(document.querySelector('#bms-state')).html("BMS State: " + bms.flag);
```

Comment:
bms.flag is an array of true and false values

Need you to convert this to a string also. The CAN doc contains it